src/librustc_back/target/x86_64_sun_solaris.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2014-2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use target::Target;
+
+pub fn target() -> Target {
+    let mut base = super::sunos_base::opts();
+    base.pre_link_args.push("-m64".to_string());
+    base.pre_link_args.push("-lsocket".to_string());
+    base.pre_link_args.push("-lposix4".to_string());
```

Comment:
Sure thing, they'll probably reside in [`rtdeps.rs`](https://github.com/rust-lang/rust/blob/master/src/libstd/rtdeps.rs).

Out of curiosity, is sunos posix? Or would a "true port" of libstd contain an entirely different implementation of `std::sys` basically (e.g. how Windows is entirely separate from Unix)